src/lib.rs

```diff
@@ -188,12 +188,12 @@
 //! }
 //! ```
 //!
-//! This example is also included in the [simulator](https://github.com/jamwaffles/embedded-graphics/tree/master/simulator/examples) crate and
+//! This example is also included in the [simulator](https://github.com/embedded-graphics/embedded-graphics/tree/master/simulator/examples) crate and
```

Comment:
This will also need to be changed after we move the examples (see comment above):
```suggestion
//! This example is also included in the [simulator](https://github.com/embedded-graphics/simulator/tree/master/examples) crate and
```